clients/src/main/java/org/apache/kafka/common/serialization/Deserializer.java

```diff
@@ -58,6 +60,13 @@ default T deserialize(String topic, Headers headers, byte[] data) {
         return deserialize(topic, data);
     }
 
+    /**
+     * Close this deserializer.
+     *
```

Comment:
If you want a new paragraph, you need to insert a `<p>` instead of an empty line.